public/js/actions/user.js

```diff
@@ -2,12 +2,24 @@
 
 var $ = require('jquery');
 
-var appActions = require('./app');
 var actionTypes = require('constants/action-types');
+var settings = require('settings');
+
+var appActions = require('./app');
+
+
+function setUpCSRF(csrfToken) {
+  console.log('setting CSRF token for subsequent requests:', csrfToken);
+  $.ajaxSetup({
+    headers: {
+      'X-CSRFToken': csrfToken,
+    },
+  });
+}
 
 
-export function signIn(accessToken) {
-  return function(dispatch) {
+export function tokenSignIn(accessToken) {
+  return dispatch => {
```

Comment:
I know the params are optional for arrow funcs with one arg, but I feel like it's more readable with them. It seems weird to me that you can have
```javascript
() = > {};
foo => {}; // <- the outlier
(foo) => {};
(foo, bar) => {};
```